src/encoded/static/components/visualizations.js

```diff
@@ -496,6 +606,335 @@ export class ChartTable extends React.Component {
     }
 }
 
+ChartList.propTypes = {
+    data: PropTypes.array.isRequired,
+    displayTitle: PropTypes.string.isRequired,
+    chartWidth: PropTypes.number.isRequired,
+    dataFilter: PropTypes.string.isRequired,
+};
+
+// Display heat map for chromatin states
+// Note: this was developed and then it was decided that this was the wrong way to visualize the data
+// Nonetheless it may be useful later
```

Comment:
This will probably not be useful in the future but I was considering leaving it just in case.

Reply:
I think I will not worry about this one for now, maybe if there are more components like this in the future.